CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - Fix #184: Make env headers comments for vagrant service-manager env @bexelbie
 - Fix #135: Refactor command.rb to make commands easier to add/maintain @budhrg
 - Adds @budhrg as co-maintainer for the plugin @navidshaikh
+- Fix #200: Simplify the eval hint for `vagrant service-manager env command` @budhrg
```

Comment:
Fix #200: Simplify the eval hint for `vagrant service-manager env` command @budhrg